packages/tooltip/src/Tooltip.js

```diff
@@ -35,11 +35,19 @@ const Container = styled.div`
     opacity: 0;
     transition: opacity 200ms ease-out;
   }
+
+  @media only screen and (max-width: 480px) {
```

Comment:
```suggestion
  @media only screen and (pointer: coarse) {
```
https://developer.mozilla.org/en-US/docs/Web/CSS/@media/pointer

Reply:
Nice, thanks!